sympy/utilities/lambdify.py

```diff
@@ -88,22 +90,74 @@
     "Matrix": "array",
     "MutableDenseMatrix": "array",
     "ImmutableMatrix": "array",
+    "ImmutableDenseMatrix": "array"
+
+}
+
+# most of the functions of scipy is similar to numpy. Checked each functions.
```

Comment:
Can you simply copy the NumPy dictionary and then update with the additional SciPy functions to reduce the repeated code?